pyasic/data/pools.py

```diff
@@ -19,15 +39,15 @@ class PoolMetrics:
     pool_stale_percent: Percentage of stale shares by the pool.
     """
 
-    accepted: int = None
```

Comment:
Probably want to keep these defaulting to none.

Reply:
Okay